.github/workflows/run-tests.yml

```diff
@@ -44,8 +44,11 @@ jobs:
       - name: Generate mocks
         run: make go_mockgen
 
-      - name: Run golangci-lint
-        run: make go_lint
+      - name: Fix imports
+        run: make gci
+
+      - name: Format the repo
+        run: make gofumpt
```

Comment:
Does this modify the code in CI (i.e. outside of version control)?

Reply:
Yes according to filters but yes

Reply:
It did previously with the `make goimports` step so I added these if this isnt intended then Ill remove them altogether

Reply:
It doesn't make sense to me for linting fixes to be applied in CI. The CI linter should fail if there are any issues, as opposed to fixing and then abandoning the changes.